src/static/css/page.css

```
@@ -277,14 +277,23 @@
 
 .authors .social {
   display: inline-block;
```

Given the longer length of this section, I think it looks better to always be on its own line rather than to the right of the author name.

```suggestion
  display: block;
```

---

Rather than this, we should just change from span to div and remove this line completed. It’s only a span for reasons below...

---

Resolved by changing to div.